apps/ledger-live-mobile/jest.config.js

```diff
@@ -47,17 +47,21 @@ module.exports = {
   ],
   testPathIgnorePatterns: ["<rootDir>/node_modules/"],
   moduleDirectories: ["node_modules"],
+  collectCoverage: true,
```

Comment:
Same for coverage as default, is it needed?

Reply:
yes sure

Reply:
deleted